### ISSUE TITLE
Having invited Grace, if she is currently registered, I would like her to receive an invitation link that allows her to join the organization.

### ISSUE THREAD

**Issue by reefdog** [PhilanthropyDataCommons/meta#38]

**Comment by kfogel** [comment]
Note similarity to #37.

**Comment by reefdog** [comment]
Yes, similar, but different! In this case, since Grace is registered (already has a PDC account), she would receive a "Join this organization?" invitation. In the case of #37, since Grace is not registered (has no PDC account), she would receive a "Sign up for the PDC [because so-and-so invited you to join their organization]" invitation. (We can talk through the UX of how this registration works -- e.g., is Grace automatically added to the inviting organization after registration, or is that a separate step after registration? -- but the key point is to distinguish these stories.)
